ffi/gn/Cargo.toml

```diff
@@ -0,0 +1,77 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_capi_gn"
+description = "GN interface to ICU4X"
+version = "0.1.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+resolver = "2"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "LICENSE"
+categories = ["internationalization"]
+# Keep this in sync with other crates unless there are exceptions
+include = [
+    "src/**/*",
+    "examples/**/*",
+    "benches/**/*",
+    "tests/**/*",
+    "include/**/*",
+    "Cargo.toml",
+    "LICENSE",
+    "README.md"
+]
+
+[package.metadata.docs.rs]
+all-features = true
+
+[dependencies]
+icu_capi = { version = "1.0", path = "icu4x_root/ffi/diplomat", default-features = false }
+icu = { version = "1.0", path = "icu4x_root/components/icu" }
+icu_provider = { version = "1.0", path = "icu4x_root/provider/core" }
+
+# [gn.package.atomic-polyfill."0.1.5"]
+# rustflags = []
+
+# [gn.package.backtrace."0.3.63"]
+# rustflags = []
+
+# [gn.package.bare-metal."0.2.5"]
+# rustflags = []
+
+# [gn.package.camino."1.0.5"]
+# rustflags = ["--cfg=path_buf_capacity"]
+
+# [gn.package.erased-serde."0.3.16"]
+# rustflags = []
+
+# [gn.package.libc."0.2.125"]
+# rustflags = ["--cfg=freebsd11", "--cfg=libc_priv_mod_use", "--cfg=libc_union", "--cfg=libc_const_size_of", "--cfg=libc_align", "--cfg=libc_int128", "--cfg=libc_core_cvoid", "--cfg=libc_packedN", "--cfg=libc_cfg_target_vendor", "--cfg=libc_non_exhaustive", "--cfg=libc_ptr_addr_of", "--cfg=libc_underscore_const_names"]
+
+[gn.package.log."0.4.14"]
```

Comment:
Is there a way to avoid having to put these `rustflags`, which may be incorrect or stale, in this Cargo.toml file here? Or at least some way to verify that they are not stale?

Reply:
Can you elaborate a bit on what such a CI script would look like? What would it invoke?